commons/ava_networks/high_level_gecko_client.go

```diff
@@ -40,6 +43,46 @@ func NewGeckoUser(username string, password string) *GeckoUser {
 }
 
 
+func (highLevelGeckoClient HighLevelGeckoClient) AddValidatorOnSubnet(
+	nodeId string,
+	pchainAddress string,
+	stakeAmount int64) error {
+	client := highLevelGeckoClient.client
+	currentPayerNonce, err := highLevelGeckoClient.getCurrentPayerNonce(pchainAddress)
+	if err != nil {
+		return stacktrace.Propagate(err, "Failed to get payer nonce from address %s", pchainAddress)
+	}
+	stakingStartTime := time.Now().Add(TIME_UNTIL_STAKING_BEGINS).Unix()
```

Comment:
Why not Now(), out of curiosity?

Reply:
@mieubrisse for a node to accept the "addValidator" call, the staking startTime must be in the future. https://docs.avax.network/v1.0/en/tutorials/adding-validators/#start-and-end-time